app/src/main/java/com/github/se/cyrcle/model/map/MapViewModel.kt

```diff
@@ -52,6 +52,11 @@ class MapViewModel : ViewModel() {
           } ?: false
         }
 
+  private val _mapScreenCoordinates =
+      MutableStateFlow<Pair<Point, Point>>(
+          Pair(Point.fromLngLat(0.0, 0.0), Point.fromLngLat(0.0, 0.0)))
```

Comment:
By experience i can tell you that it is better to use a determined location based on a constant value rather than the (0,0) point which has the tendency of creating unexpected behavior in certain case.

Reply:
Yes, I saw at first that no parkings were fetched because of this, I managed to find a fix around this since I found it a bit arbitrary. I nevertheless added a default point in case.

Reply:
Congrats then ! No need to change the point if you managed to handle this.